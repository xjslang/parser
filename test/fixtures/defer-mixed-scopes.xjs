// Valid: defer inside regular function
function test1() {
  defer { console.log('ok') }
}

// Invalid: defer in module level
defer {
  console.log('should fail')
}

test1()
