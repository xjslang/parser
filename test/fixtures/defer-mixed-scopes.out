defer statements are not allowed at top-level
